src/github/pullRequestModel.ts

```diff
@@ -278,6 +279,41 @@ export class PullRequestModel extends IssueModel<PullRequest> implements IPullRe
 		}
 	}
 
+	async updateMilestone(id: string): Promise<string> {
+		const { mutate, schema } = await this.githubRepository.ensure();
+
+		const finalId = id === 'null' ? null : id;
+
+		const { data } = await mutate<UpdateMilestone>({
```

Comment:
since this is using the `UpdatePullRequest` mutation, I think this should be `mutate<UpdatePullRequestResponse>`. The data being returned still looks the same, so `UpdateMilestone` can be removed